docs/section18/Rule_18-2.md

```diff
@@ -0,0 +1,80 @@
+- use the function get_building_total_lab_exhaust_from_zone_exhaust_fan to get the totat lab exhaust air flow from zone exhaust fans.  This will be used when checking for laboratory systems: `lab_zone_exhaust = get_building_total_lab_exhaust_from_zone_exhaust_fan(P_RMI)`
+- create a list of laboratory zones meeting G3_1_1d: `laboratory_zones_meeting_G3_1_1d = [x for x in target_baseline_systems if target_baseline_systems[x]["SYSTEM_ORIGIN"] == G3_1_1d]`
+- evaluate the rule for each HVAC system in the B_RMI: `for hvac_system in B_RMI...heating_ventilating_air_conditioning_systems:`
+	- set the result to FAIL: `result = FAIL`
+	- find the hvac_system type by looking through the baseline_system_types_dict for the hvac_system.id: `for hvac_system_type in baseline_system_types_dict:`
+		- find the actual system type of the system (for example, SYS_5 instead of SYS_5B).  Start by looping through the possible system types: `for possible_type in [SYS_1,SYS_2,SYS_3,SYS_4,SYS_5,SYS_6,SYS_7,SYS_8,SYS_9,SYS_10,SYS_11,SYS_12,SYS_13):`
+			- set system_type to possible_type if the result of using the baseline_system_type_compare function is TRUE: `if baseline_system_type_compare(hvac_system_type,possible_type,FALSE): system_type = possible_type`
+		- see if the hvac_system.id is in the list: `if hvac_system.id in? baseline_system_types_dict[hvac_system_type]:`
+  			- Get the list of zones served by the system from zones_and_terminal_unit_list_dict: `zones_served_by_system = zones_and_terminal_unit_list_dict[hvac_system.id]["ZONE_LIST"]`
+     			- check if the system is a single-zone system type: `if any(system_type == sys_type for sys_type in [SYS_1,SYS_2,SYS_3,SYS_4,SYS_9,SYS_10,SYS_11,SYS_12,SYS_13]):`
+     				- only systems that serve single zones can be categorized as single zone systems, so set result to pass: `result = PASS`
+     			- otherwise it is a multi-zone system:`else:`
+					- for a multi-zone system, in a building without lab zones, the system can pass if:
+						1. all zones are on the same floor
+						2. there are no other systems of the same type that serve zones on the same floor
+					- get the list of zones on the same floor by using the function get_zones_on_same_floor: `zones_on_floor = get_zones_on_same_floor(B_RMI,zones_served_by_system[0])`
+					- check if all the zones served by the system are on the same floor: `if all(zone in zones_on_floor for zone in zones_served_by_system):`
+						- now check if there are any other systems of the same system type that serve zones on this floor.  First loop through baseline_system_types_dict again: `for hvac_system_type2 in baseline_system_types_dict:`
+							- check if the hvac_system_type2 is the same type: `if baseline_system_type_compare(hvac_system_type2,system_type,FALSE):`
```

Comment:
Should we use baseline_system_type_compare(TRUE) here? Otherwise, we'll not allow systems of the same type but with different letters (e.g., 5 and 5a) on the same floor.